turbinia/workers/docker.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2015 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Task for running docker-explorer."""
+
+from __future__ import unicode_literals
+
+import json
+import logging
+import os
+import subprocess
+
+from turbinia import TurbiniaException
+from turbinia.evidence import DockerContainer
+from turbinia.workers import Priority
+from turbinia.workers import TurbiniaTask
+
+log = logging.getLogger('turbinia')
+
+
+class DockerContainersEnumerationTask(TurbiniaTask):
```

Comment:
Can we also add a test here?  It should be relatively easy to test Tasks now, there is a `TestTurbiniaTaskBase` that you can inherit from and pass in the right evidence, etc.  Here's an example:
https://github.com/google/turbinia/blob/master/turbinia/workers/bulk_extractor_test.py

Reply:
Done

Reply:
Thanks!